src/features/island/hud/components/settings-menu/linked-accounts/LinkedAccounts.tsx

```
@@ -153,9 +162,21 @@ export const LinkedAccounts: React.FC<ContentComponentProps> = ({
   const fslId = useSelector(gameService, _fslId);
   const oauthNonce = useSelector(gameService, _oauthNonce);
+  const lastUnlink = useSelector(gameService, _lastUnlink);
 
   const [revealWallet, setRevealWallet] = useState(false);
 
+  // "Unlinked. Can be linked again after {date}" for the provider that was
+  // just unlinked, while its lock is still running.
+  const unlinkedUntil = (provider: UnlinkableSocialProvider) =>
+    lastUnlink?.provider === provider &&
+    !!lastUnlink.availableAt &&
+    lastUnlink.availableAt > Date.now()
```

ESLint

[error] Error: Cannot call impure function during render

`Date.now` is an impure function. Calling an impure function can produce unstable results that update unpredictably when the component happens to re-render. (https://react.dev/reference/rules/components-and-hooks-must-be-pure#components-and-hooks-must-be-idempotent).

/home/jailuser/git/src/features/island/hud/components/settings-menu/linked-accounts/LinkedAccounts.tsx:174:30
  172 |     lastUnlink?.provider === provider &&
  173 |     !!lastUnlink.availableAt &&
> 174 |     lastUnlink.availableAt > Date.now()
      |                              ^^^^^^^^^^ Cannot call impure function
  175 |       ? t("linkedAccounts.subtext.unlinkedUntil", {
  176 |           date: formatAvailableAt(lastUnlink.availableAt),
  177 |         })

(react-hooks/purity)